doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/SiteRenderingContext.java

```diff
@@ -388,4 +391,20 @@ public File getRootDirectory() {
     public void setRootDirectory(File rootDirectory) {
         this.rootDirectory = rootDirectory;
     }
+
+    /**
+     *
+     * @return the retriever for parser configurations (only relevant for Doxia sources)
+     * @since 2.0.0
+     */
+    public ParserConfigurationRetriever getParserConfigurationRetriever() {
+        if (parserConfigurationRetriever == null) {
```

Comment:
Am I stupid or why not just return the object?

Reply:
because this is not supposed to return null ever (will clarify in the javadoc).